### PR TITLE
Documentation rework - Phase 1

### PR DESCRIPTION
**Pull request by cfusterbarcelo**
# Documentation Rework

## Changes
- Four sections on the sidebar: Getting started, Guides, Tools and Resources, Help Desk and Terms of Service. All files have been moved to follow this structure.
- Old, unused and empty files have been moved to /docs/deprecated
- New sidebar do not use pointers from `index.html`

## Phases
Phase 1 includes adding some missing information and move the files in a sensible way. Next phase will be to add new missing documentation and rework some of the existing files.

### CODE REVIEW

**Comment by cfusterbarcelo** [comment]
Somehow is not working and the documentation not showing. I'll take care of it.

**Comment by oeway** [comment]
@carlosuc3m Is this PR ready to merged?

File: docs/guides/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Guides
+The BioImage Model Zoo is intended to be used by different roles in the community as stated in [How to participate in the BioImage Model Zoo](/getting_started/README.md). This section contains guides for the different roles in the community.
+
+* **[User Guide](user_guide.md)**: Guides for users willing to download and use the models and other resources from the BioImage Model Zoo
+* **[Developers Guide](developer_guide.md)**: Guides for who is willing to contribute to the BioImage Model Zoo with models, applications or softwares.
```
Comment (oeway) [inline docs/guides/README.md]:
```suggestion
* **[Developers Guide](guides/developer_guide.md)**: Guides for who is willing to contribute to the BioImage Model Zoo with models, applications or softwares.
```

**Comment by cfusterbarcelo** [comment]
This PR is ready to deploy! 🚀🎉 
@oeway : Could you resolve conflicts and merge...?

**Comment by oeway** [comment]
Hi @cfusterbarcelo I just merged the changes, however, somehow the history disappeared from the commit history, sorry about that! If you have a local copy of it, feel free to merge with the remote copy in your fork, them push it again so we can restore the commit history.

I tried to render the docs website locally, it appears that the docs side bar are not working, and there are still many broken image links, could you take a look? We should also remove the deprecated folder if that's not needed, we don't make copies of these files, instead, we recover them from github history if needed. Are you sure that all the deprecated files are not need anymore?

**Comment by oeway** [comment]
Looks good now, it wasn't working for some reason. I will merge and we will take it from there!

**Pull request merged**